lib/handler.js

```diff
@@ -60,3 +60,5 @@ exports = module.exports = function createStrongErrorHandler(options) {
     }
   };
 };
+
+module.exports.buildResponseData = buildResponseData;
```

Comment:
This should be exported from `/lib/data-builder.js` and not this file which is just importing in the data-builder.